code/modules/projectiles/guns/specialist.dm

```diff
@@ -556,26 +572,23 @@ Note that this means that snipers will have a slowdown of 3, due to the scope
 	placed_overlay_iconstate = "sadar"
 	windup_delay = 0.4 SECONDS
 	///the smoke effect after firing
-	var/datum/effect_system/smoke_spread/smoke
+	var/obj/effect/abstract/particle_holder/backblast
 	///removes backblast damage if false
 	var/backblastdamage = TRUE
 
-/obj/item/weapon/gun/launcher/rocket/Initialize(mapload, spawn_empty)
-	. = ..()
-	smoke = new(src, FALSE)
-
-/obj/item/weapon/gun/launcher/rocket/Destroy()
-	QDEL_NULL(smoke)
-	return ..()
-
 //Adding in the rocket backblast. The tile behind the specialist gets blasted hard enough to down and slightly wound anyone
 /obj/item/weapon/gun/launcher/rocket/apply_gun_modifiers(obj/projectile/projectile_to_fire, atom/target)
 	. = ..()
 	var/turf/blast_source = get_turf(src)
 	var/thrown_dir = REVERSE_DIR(get_dir(blast_source, target))
 	var/turf/backblast_loc = get_step(blast_source, thrown_dir)
-	smoke.set_up(0, backblast_loc)
-	smoke.start()
+	var/angle = Get_Angle(loc, target)
+	var/x_component = sin(angle) * -30
+	var/y_component = cos(angle) * -30
+	backblast = new(blast_source, /particles/backblast)
+	backblast.particles.velocity = list(x_component * 1, y_component * 1)
+	QDEL_IN(backblast, 0.6 SECONDS)
+
```

Comment:
Should use QDEL_NULL_IN or even better just add the particle holder to vis contents as needed so you dont need to recreate it every time